config/manager/kustomization.yaml

```diff
@@ -1,2 +1,16 @@
 resources:
 - manager.yaml
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- files:
+  - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: quay.io/ykarel/neutron-operator
```

Comment:
don't think you wanted to add this from your latest pushes?

Reply:
yes right, my bad, updating